### DISCUSSION TITLE
How to modify the categories of the coco dataset?

### DISCUSSION THREAD

**Discussion by MACHUNHAI** [open-mmlab/mmyolo#739] [Q&A]
在mmyolo中训练yolov8时，使用的是数据集是自己的coco格式的数据集（在mmdetection中可用），训练之后跑测试，发现检测出的类别还只是标准的coco数据集的类别，请问大佬如何修改coco数据集的类别，需要把coco数据集重新做成yolo格式的吗？目前的目录如下：
<img width="920" alt="image" src="https://user-images.githubusercontent.com/43568114/233109698-2e9e5c04-b75d-48db-adaa-904fcabe00dd.png">

**Comment by josh3255** [comment]
If you initialized the model with init_detector function as follows

```python
model = init_detector(args.config, args.checkpoint, device=args.device)
```
try adding the code as below

```python
model.dataset_meta['classes'] = ('your', 'custom', 'classes')
```
